recipes/komplot/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "komplot" %}
+{% set version = "0.0.1" %}
+{% set python_min = "3.9" %}
```

Comment:
This only needs to be set if it should be larger than the default value of "3.10".

Reply:
Thanks for the fast response to the review request. The line has been removed as recommended.